src/components/ContactForm/ContactForm.tsx

```diff
@@ -23,8 +24,12 @@ export default function ContactForm({
     runAsync(async () => {
       try {
         // Handle submit actions
-        // console.log('Handle submit', values)
-        showSuccessMsg()
+        apiClient.saveContactRequest({
```

Comment:
**nit**: you should be able to call server action here, let's do this instead of calling endpoint